spec/system/admin/products_v3/products_spec.rb

```diff
@@ -1060,23 +1196,30 @@
       let(:default_variant_selector) {
         "tr:has(input[aria-label=Price][value='#{product_a.price}'])"
       }
+      let(:link_edit_variant) {
+        "/admin/products/#{Spree::Product.first.id}/variants/#{Spree::Variant.first.id}/edit"
+      }
 
       describe "Actions columns (delete)" do
         before do
           visit admin_products_url
         end
 
         it "shows an actions menu with a delete link when clicking on icon for product. " \
+           "shows link to clone and edit the product;" \
            "doesn't show delete link for the single variant" do
           within product_selector do
             page.find(".vertical-ellipsis-menu").click
+            expect(page).to have_link "Clone", href: spree.clone_admin_product_path(product_a)
+            expect(page).to have_link "Edit", href: spree.edit_admin_product_path(product_a)
```

Comment:
I think these are already tested in the "edit" and "clone" sections above, so probably aren't necessary here.

Reply:
Thanks @dacook. I've removed those two assertions, and squashed that commit.